pkg/ebpf/c/types.h

```diff
@@ -353,13 +359,14 @@ typedef struct event_data {
     u64 param_types;
 } event_data_t;
 
-// A control plane signal - sent to indicate some critical event which should be processed
-// with priority.
+// Control Plane Signal
+//
+// Higher priority (and, most of the times, simpler) events to be processed faster by userland.
 //
-// Signals currently consist of shortened events sent only with their arguments.
-// As such, they consist of an event id and an argument buffer.
-// If we ever require a signal independent of an event, the event_id field should change
-// accordingly.
+// Signals currently consist of shortened events sent only with their arguments. As such, they
+// consist of an event id and an argument buffer. If we ever require a signal independent of an
```

Comment:
I believe that now we do, considering the exec signal (right?) has an additional argument not included in the event itself.

Reply:
True. The idea of adding extra args just for the plane came later.